GITHUB_ORG_MANAGEMENT_POLICY.md

```diff
@@ -36,6 +36,9 @@ The following groups are granted Ownership permissions:
 * **OpenJS Director of Program Management** The OpenJS Director of Program
   Management will limit their use of the access granted to:
     * accepting GitHub terms and conditions.
+  They will be added to the team called openjs_administrative_support as we
```

Comment:
I think most of the team names are using hyphens as separators.

```suggestion
  They will be added to the team called [openjs-administrative-support](https://github.com/orgs/nodejs/teams/openjs-administrative-support) as we
```

(also link to a potential team page to be created)